src/app/tech/page.tsx

```diff
@@ -1,14 +1,13 @@
+import HeroSection from "@/components/tech/HeroSection";
 import PageTitle from "../../components/pageTitle";
 import Tech from "./tech";
```

Comment:
**nitpick (blocking):** Remove unused imports.  If we're no longer using those components due to the re-design then we can delete the files.